script.user.js

```diff
@@ -50,6 +50,8 @@
 // @icon        https://github.com/tophf/mpiv/raw/master/icon.png
 // ==/UserScript==
 
+import mpiv from './types';
```

Comment:
`import` cannot be used outside of modules. Userscripts aren't modules and their code isn't compiled so this will break the script.

Reply:
Ok, I replaced export/import with a single line in the script: `/// <reference types='./types' />`
and eventually a jsconfig.json (using `"typeAcquisition"` and `"include"` pointing to types.d.ts )